.idea/codeStyles/Project.xml

```diff
@@ -1,5 +1,23 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
```

Comment:
rm (why is this file checked in?)

Reply:
The file is fine, but these changes should be reverted.